flatdata-rs/README.md

```diff
@@ -1,20 +1,6 @@
-# flatdata-rs [![travis status]][travis] [![latest version]][crates.io] [![docs]][docs.rs]
+# flatdata-rs [![latest version]][crates.io] [![docs]][docs.rs]
```

Comment:
Note: Need to adjust the path in the toml definition of the crate.

Reply:
what do you mean?

Reply:
I mean that `Cargo.toml` contains a path to the repository, which has now changed. (Just commented on this file due to the lack of context.)